packages/@okta/vuepress-site/docs/guides/single-logout/main/index.md

```diff
@@ -1,127 +1,48 @@
-
-**Event 3**
-
-Because Apps 2 and 3 have user sessions in other browsers, and on other devices, the apps may terminate these sessions from the server side. When the user tries to use these apps in the respective browsers, the user discovers that the apps have invalidated the user’s browser sessions.
-
-Downstream SAML apps terminate a specific session associated with the user or terminate all sessions associated with the user. This depends on whether `sessionIndex` (SAML) is included in the IdP-initiated logout request. For OIDC apps, this depends on whether the session ID (`sid`) and issuer (`iss`) are included.
-
-## Configure SLO
-
-SLO supports SAML apps, and web and single-page (SPA) OIDC apps. The following steps explain how to configure your apps for SLO.
-
-### Update your <StackSnippet snippet="apptype" inline /> app integration to use SLO
-
-Use the following steps to update your <StackSnippet snippet="apptype" inline /> app to use SLO:
-
-1. In the Admin Console, go to **Applications** > **Applications**.
-2. Select the <StackSnippet snippet="apptype" inline /> app that you want to update to use SLO.
-3. On the **General** tab, in the <StackSnippet snippet="sectionname" inline /> section, click **Edit**.
-
-<StackSnippet snippet="configureslo" />
-
-### Use the API to update your <StackSnippet snippet="apptype" inline /> app integration
-
-The following example shows you how to use the API to update your <StackSnippet snippet="apptypeapi" inline /> integration for SLO. <StackSnippet snippet="apispecific" inline />
-
-<StackSnippet snippet="participateslonote" />
-
-1. Send a GET app request and copy the response body for use in the PUT request.
-
-    <StackSnippet snippet="getrequest" />
-
-2. Update the response body by editing the <StackSnippet snippet="object" inline /> object:
-
-    <StackSnippet snippet="properties" />
-    <br>
-
-    **Example request**
-
-    <StackSnippet snippet="body" />
-
-3. Use the updated response body in a PUT request.
-
-    <StackSnippet snippet="putrequest" />
-    <br>
-
-    **Example response**
-
-    <StackSnippet snippet="response" />
+<StackSnippet snippet="updateapporidp" />
 
 ## Events
 
-After Okta initiates the outbound logout request to downstream apps, Okta includes the number of OIDC and SAML app logouts that occurred with SLO. Those numbers are found in the System Log event `user.session.end` under `DebugData`:
+After Okta initiates the outbound logout request to <StackSnippet snippet="downup" inline />, Okta includes the number of OIDC and SAML app logouts that occurred with SLO. Those numbers are found in the System Log event `user.session.end` under `DebugData`:
 
 * `TotalOidcLogoutRequests`: Lists the total number of logout requests for OIDC apps
 * `TotalSamlLogoutRequests`: Lists the total number of logout requests for SAML apps
```

Comment:
if IdP SLO feature is enabled, these counts will be for OIDC apps + idps, and SAML apps + idps